src/SimpleCalculator.Web/Controllers/HomeController.cs

```diff
@@ -29,20 +34,44 @@ public IActionResult Index()
     public IActionResult Add(CalculationInputModel model)
     {
         model.Result = _addition.Add(model.FirstNumber, model.SecondNumber);
+        var calculatorMemory = new CalculatorMemory()
```

Comment:
nitpick: could you please take a look at [AutoMapper](https://automapper.org/) framework? It's a good practice to have an independent conversion logic